### DISCUSSION TITLE
Grid thumbnails, putting photos first for the user

### DISCUSSION THREAD

**Discussion by rightsaidfred99** [photoprism/photoprism#1530] [Ideas]
Hi there, 

Stumbled upon this project and it looks great! One thing I noticed with the demo UI - is that there seemed to be a focus on metadata rather than the photos itself when presented with the UI on first load. 

Just wondering, if there is a "timeline" type view or larger thumb nailed grid (where photos aren't cropped) which would put the photos centre and front for the user? 

I think it help new users to the platform but also for photographers who might not like seeing 1:1 cropped thumbnails in a view. Just my opinion I guess.

**Comment by graciousgrey** [comment]
Currently we have the "mosaic" view to show photos without metadata information (https://docs.photoprism.org/user-guide/organize/views/). A timeline view is already on our roadmap https://github.com/photoprism/photoprism/issues/152 :)

**Comment by rightsaidfred99** [comment]
Ah thanks, what I mean though with the "mosaic" view is that, just as a feature request, it would be awesome to have the dynamic masonry grid. Where it doesn't crop photographs is all. :)

Kind of like Wordpresses' envira gallery, google photos etc...

**Reply by lastzero** [reply]
see https://github.com/photoprism/photoprism/issues/85
